src/status_im2/constants.cljs

```diff
@@ -185,6 +185,7 @@
 (def regx-address #"^0x[a-fA-F0-9]{40}$")
 (def regx-address-contains #"(?i)0x[a-fA-F0-9]{40}")
 (def regx-starts-with-uuid #"^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}")
+(def regx-address-fragment #"^0x[a-fA-F0-9]{1,40}$")
```

Comment:
imo `fragment` does not offer so much info here. could there be a clearer name? 🤨

Reply:
perhaps even a comment with an example beside it would help 👍

Reply:
Sure, this regex evaluates wether if the string is a full or partial address, maybe something like `regx-full-or-partial-address`?